presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -65,10 +75,8 @@ public void testCteExecutionWhereOneCteRemovedBySimplifyEmptyInputRule()
                 " b AS ((SELECT orderkey FROM t) UNION (SELECT orderkey FROM t1)) " +
                 "SELECT * FROM b";
         QueryRunner queryRunner = getQueryRunner();
-        compareResults(queryRunner.execute(getMaterializedSession(),
-                        sql),
-                queryRunner.execute(getSession(),
-                        sql));
+        verifyResults(queryRunner, sql, ImmutableList.of(generateMaterializedCTEInformation("t", 1, false, false),
```

Comment:
@jaystarshot - please note that the is_materialized value for the cte named t is false.